libs/client/cm/data-access/src/lib/pois/pois.effects.ts

```diff
@@ -124,7 +126,7 @@ export class PoiEffects {
             this.status.presentAlertWithAction({
               alertData: {
                 title: `Confirm Withdrawal`,
-                description: `This withdraws and <b>permanently deletes</b> your POI to <b>${poi.enrollment.project.title}</b>.`,
+                description: `This withdraws and <b>permanently deletes</b> your POI to <b>${poi.enrollments[0].project.title}</b>.`,

```

Comment:
We may need to update this to have two different messages depending on whether or not there is only one enrollment tied to the poi. If only one, the current text would be correct, but if more than one, it would be more appropriate to say that you will be removed from this poi.